brushfire-finatra/src/main/scala/com/stripe/brushfire/finatra/BrushfireServer.scala

```diff
@@ -7,7 +7,7 @@ import com.twitter.finagle.http._
 import com.twitter.bijection._
 
 class BrushfireServer extends FinatraServer {
-  def score[K, V, T: Semigroup, P](root: String, trees: Iterable[Tree[K, V, T]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
+  def score[K, V, T: Semigroup, P](root: String, trees: Iterable[Tree[K, V, T, Unit]], voter: Voter[T, P])(fn: ParamMap => Map[K, V]) = {
```

Comment:
If the annotations aren't used by the server, a `Unit` annotation will parse successfully but will use less memory than `A` (whatever it is) at runtime... question is: are annotations useful for a model server?